app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -101,6 +102,8 @@ def check_password
     return if !@assessment.password_protected? || can?(:manage, @assessment)
 
     unless authentication_service.authenticated?
+      log_service.log_submission_access(request)
```

Comment:
> If new submission or correct password, the 2 session IDs will be the same.
If this is always the case I think no need a column, otherwise better to have. we can display failed attempts in a red background then 😄